assets/src/edit-story/components/panels/textStyle/font.js

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { useCallback } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { InputGroup, SelectMenu } from '../../form';
+import { MAX_FONT_SIZE, MIN_FONT_SIZE } from '../../../constants';
+import { useFont } from '../../../app/font';
+
+function FontControls({ properties, state, setState }) {
+  const { fontFamily, fontWeight, fontSize } = properties;
+  const {
+    state: { fonts },
+    actions: { getFontWeight, getFontFallback },
+  } = useFont();
+
+  const handleNumberChange = useCallback(
+    (property) => (value) =>
+      setState({ ...state, [property]: parseInt(value) }),
+    [setState, state]
+  );
+  return (
+    <>
+      {fonts && (
+        <SelectMenu
+          label={__('Font family', 'web-stories')}
+          options={fonts}
+          value={state.fontFamily}
+          isMultiple={fontFamily === ''}
+          onChange={(value) => {
+            const currentFontWeights = getFontWeight(value);
+            const currentFontFallback = getFontFallback(value);
+            const fontWeightsArr = currentFontWeights.map(
+              ({ thisValue }) => thisValue
+            );
+            const newFontWeight =
+              fontWeightsArr && fontWeightsArr.includes(state.fontWeight)
+                ? state.fontWeight
+                : 400;
```

Comment:
Not sure about the impact, but `400` will not work for 6 fonts in the current set (is will setState 400, but option will be e.g. 300/Light):
![image](https://user-images.githubusercontent.com/21036224/75462804-140adc80-5985-11ea-8d73-78fcb296fc45.png)

Reply:
Perhaps choosing the first available font weight if `400` is not available. Thoughts?

Reply:
Makes sense to me.

Reply:
Alternative (overkill) is to select the most similar weight, so from `[100,200,300]` get `300` and so on, but with our current set of fonts, the first available option will work just fine.

_Watch out for Molle font (empty array)._

Reply:
Addressed in 588807a